visallo/visallo
# GraphWorkProductService performance improvements

1.) Added `FetchHint.EDGE_REFS` when loading vertices since they are only used to find edges
2.) Removed the `trimCompoundNodes` as that functionality isn't needed due to changes elsewhere.


Testing Instructions:
1.) Load a workspace with several vertices
2.) Collapse the vertices
3.) Refresh the page and ensure vertices are still collapsed
4.) Uncollapse the vertices
5.) Use undo to make the vertices collapsed again
6.) Refresh the page and ensure vertices are still collapsed
7.) Uncollapse the vertices
8). Use undo to make the vertices collapsed again
9.) Use redo to make the vertices uncollapsed again
10.) Refresh the page and ensure vertices are not collapsed
